README.md

```diff
@@ -9,59 +7,70 @@ This is a Couchbase connector node module for [Loopback](http://loopback.io/) wi
 ### Install
 
 ```
-npm install loopback-connector-couchbase3 --save
+npm install loopback-connector-couchbaseX --save
 ```
 
-### Config
+### DataSource Config in LoopBack
 
+- For Couchbase version >= 5, as default
 ```
 # datasources.json
 {
-  "couchbaseTestBucket": {
-    "name": "couchbaseTestBucket",
-    "connector": "couchbase3",
+  "testDs": {
+    "name": "testDs",
+    "connector": "couchbase5",
+    "version": 5,  // optional
     "cluster": {
       "url": "couchbase://localhost",
+      "username": "username",
+      "password": "password",
       "options": {}
     },
     "bucket": {
-      "name": "test_bucket",
-      "password": ""
+      "name": "test_bucket"
     }
   }
 }
 ```
 
-## Dev
-
+- For Couchbase version < 5
 ```
-export COUCHBASE="couchbase4"
-export COUCHBASE_USER="Administrator"
-export COUCHBASE_PASS="password"
-./dockers/start-couchbase.sh
+# datasources.json
+{
+  "testDs": {
+    "name": "testDs",
+    "connector": "couchbase5",
+    "version": 3,
+    "cluster": {
+      "url": "couchbase://localhost",
+      "options": {}
+    },
+    "bucket": {
+      "name": "test_bucket",
+      "password": ""
+    }
+  }
+}
 ```
 
 ## Test
 
-precheck: couchbase server is runing with two buckets named `test_bucket`, `test_ping` and enabled its flush feature.
-run: `npm test`
+`yarn test`
 
-## Git Summary
+## Summary
 
-```
- project  : loopback-connector-couchbase3
- repo age : 2 years, 2 months
- active   : 53 days
- commits  : 106
- files    : 38
- authors  :
-    59  Makara Wang        55.7%
-    25  CCharlieLi         23.6%
-     7  chopperlee         6.6%
-     4  Xavier Zhou        3.8%
-     3  Leo Zhou           2.8%
-     3  wwayne             2.8%
-     2  greenkeeperio-bot  1.9%
-     2  xavier             1.9%
-     1  Marc Bachmann      0.9%
-```
+project  : loopback-connector-couchbaseX
```

Comment:
could you db-check the format of `Summary`, without the code snippet `backtick` the format might be messy.